pymc/distributions/mixture.py

```diff
@@ -817,15 +817,19 @@ def _hurdle_mixture(*, name, nonzero_p, nonzero_dist, dtype, **kwargs):
 
     nonzero_p = pt.as_tensor_variable(nonzero_p)
     weights = pt.stack([1 - nonzero_p, nonzero_p], axis=-1)
+
+    truncated_kwargs = {k: v for k, v in kwargs.items() if k in {"lower", "upper", "max_n_steps"}}
```

Comment:
upper is not supposed to be part of hurdle, so I would just do max_n_steps, which can be an explicit kwarg in the signature?

Reply:
Good point